tests/utils.go

```diff
@@ -787,8 +787,13 @@ func AdjustKubeVirtResource() {
 	PanicOnError(err)
 
 	kv := GetCurrentKv(virtClient)
+
 	originalKV = kv.DeepCopy()
 
+	if !flags.ApplyDefaulte2eConfiguration {
```

Comment:
Should this check be moved down after `kv.Spec.CertificateRotationStrategy`? Or is that not important?

Reply:
I was thinking about it. I think this is another disruptive setting which we just blindly applied. I am not entirely sure if it should go behind this flag, or if  it deserves an extra flag.